### ISSUE TITLE
Sparse -> Dense conversion

### ISSUE THREAD

**Issue by milankl** [fgerick/Limace.jl#11]
(Hi @fgerick just opening some issues for the JOSS review. I'll tag those later once I'm assigned there.)

Could you clarify why it's necessary to convert to a dense matrix after you construct a sparse one? Is it not possible to do the LinearAlgebra based on the sparse? That should be important for memory and performance.

**Comment by fgerick** [comment]
Hi @milankl, 
The conversion from sparse to dense matrices is only necessary when we compute the full spectrum of modes. It is not necessary (and also not feasible) to do this for larger scale problems. There are some examples at low resolution (small truncation degree `N`) that compute the dense spectrum, but `Limace.jl` also provides a sparse method to compute only a few eigensolutions close to a target (see the example on Torsional Alfvén modes). Do you think it is necessary to highlight this somewhere in more detail?

**Comment by milankl** [comment]
Yeah that would be great, have a little paragraph on that somewhere in the docs and outline when to densify when not?

**Comment by milankl** [comment]
Picking this up again ☝ I think there should be somewhere a discussion around sparse/dense. I think SparseArrays.jl has some missing functionality around eigenvalues problems, if that's the case and therefore a user has to go to dense that's fine, but just to highlight when that's needed would be good!!

https://github.com/openjournals/joss-reviews/issues/7593

**Comment by fgerick** [comment]
Now discussed in the [Solving the eigenvalue problem](https://fgerick.github.io/Limace.jl/dev/eigen/#Solving-the-eigenvalue-problem) section.

**Comment by milankl** [comment]
This looks great Felix! Thanks for including this in the documentation!
